CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ set(${PROJECT_NAME}_VERSION "${${PROJECT_NAME}_VERSION_MAJOR}.${${PROJECT_NAME}_
 # the base path will be used, so watch for incompatibilities!!
 #
 set(CMAKE_MODULE_PATH
-    ${PROJECT_SOURCE_DIR}/cmake/Modules
+    ${Geant4_SOURCE_DIR}/cmake/Modules
```

Comment:
I'd rather see use of `PROJECT_{SOURCE,BINARY}_DIR` as this is consistent throughout. Scoping to the explicit project name inside said project means that's more difficult to change later on. As @jrmadsen has noted, there are places where this is needed to avoid referring to parent projects.

There are lots of changes of this form, so I won't comment/review every one of them, but they should _all_ be changed from `Geant4_{SOURCE,BINARY}_DIR` to the `PROJECT_..` equivalent.